extide/gradle/netbeans-gradle-tooling/src/main/java/org/netbeans/modules/gradle/tooling/NbProjectInfoBuilder.java

```diff
@@ -627,7 +650,7 @@ private void collectModuleDependencies(final NbProjectInfoModel model, String co
     private Set<Configuration> configurationsToSave() {
         return project
                 .getConfigurations()
-                .matching(c -> !CONFIG_EXCLUDES.contains(c.getName()))
+                .matching(c -> !CONFIG_EXCLUDES_PATTERN.matcher(c.getName()).matches())
```

Comment:
The added regexp excludes all `*DependencyMetadata` configurations like `compileOnlyDependencyMetadata` used by Kotlin plugin. Useless for the IDE, but since the module has no JVM artifacts, it confuses the caching / validation mechanism.